ibis/backends/tests/test_export.py

```diff
@@ -456,3 +446,130 @@ def test_empty_memtable(backend, con):
     table = ibis.memtable(expected)
     result = con.execute(table)
     backend.assert_frame_equal(result, expected)
+
+
+@pytest.mark.notimpl(["dask", "druid"])
+@pytest.mark.notimpl(
+    ["impala"], raises=AttributeError, reason="missing `fetchmany` on the cursor"
+)
+@pytest.mark.skipif(pa.__version__ < "12.", reason="Pyarrow >= 12 required")
+class TestDataframeInterchange:
```

Comment:
In other projects I've started to group similar tests into test classes. I like this method for keeping the test files organized, and have found it to be effective even for large and evolving test files. Otherwise I've found pytest test cases can end up scattered throughout the test file, making it harder to see what tests exist for a specific piece of functionality.

I do recognize this is a new pattern in ibis though - happy to move these to functions alone if needed.

Reply:
I'd rather keep these as functions, but you can achieve the same level of organization by making a new test module, which is even easier to discover since you don't even have to open a file to know what's inside!